src/app/static/src/app/components/ErrorReport.vue

```diff
@@ -0,0 +1,130 @@
+    <modal :open="open">
+        <h4 v-translate="'reportIssues'"></h4>
+        <form class="form">
+            <div class="form-row">
+                <div class="col-4">
+                    <label for="section" v-translate="'section'"></label>
+                </div>
+                <div class="col-8">
+                    <select class="form-control"
+                            v-model="currentSection"
+                            id="section">
+                        <option v-for="step in steps"
+                                :key="step.number"
+                                v-translate:value="step.textKey"
+                                v-translate="step.textKey">
+                        </option>
+                    </select>
+                </div>
+            </div>
+            <div class="form-group">
+                <label for="description"
+                       class="mb-0"
+                       v-translate="'description'"></label>
+                <div class="small text-muted" v-translate="'errorDescriptionHelp'"></div>
+                <textarea id="description"
+                          v-model="description"
+                          class="form-control"></textarea>
+            </div>
+            <div class="form-group">
+                <label for="reproduce"
+                       class="mb-0"
+                       v-translate="'reproduce'"></label>
+                <div class="small text-muted" v-translate="'reproduceErrorHelp'"></div>
+                <textarea id="reproduce"
+                          v-model="reproduce"
+                          class="form-control"></textarea>
+            </div>
+        </form>
+        <template v-slot:footer>
+            <button type="button"
+                    class="btn btn-red"
+                    @click="sendErrorReport"
+                    v-translate="'send'">
+            </button>
+            <button type="button"
+                    class="btn btn-white"
+                    @click="cancelErrorReport"
+                    v-translate="'cancel'">
+            </button>
+        </template>
+    </modal>
+</template>
+<script lang="ts">
+    import Vue from "vue"
+    import {mapStateProp} from "../utils";
+    import {StepDescription, StepperState} from "../store/stepper/stepper";
+    import {RootState} from "../root";
+    import {Language} from "../store/translations/locales";
+    import i18next from "i18next";
+    import Modal from "./Modal.vue";
+
+    interface Data {
+        description: string
+        reproduce: string
+        section: string
+    }
+
+    interface Methods {
+        sendErrorReport: () => void
+        cancelErrorReport: () => void
+        resetData: () => void
+    }
+
+    interface Computed {
+        currentLanguage: Language
+        currentSectionKey: string
+        currentSection: string
+        steps: StepDescription[]
+    }
+
+    export default Vue.extend<Data, Methods, Computed, "open">({
```

Comment:
I had never seen a literal prop defined in the this way! At first glance I was rather wary of it introducing an untyped prop, but it's actually consistent with relevant prop of the Modal component.

Reply:
You're right, this is a little lazy and does lose the type info. I'll actually change it over to a regular typed prop.